roles/var_files_present/tasks/main.yml

```diff
@@ -54,5 +70,10 @@
     msg: "The permissions on the added file are incorrect"
   when: added_file.stat.mode != "0644"
 
-- name: Check if the SELinux context on the added file is correct
+- name: Check for svirt_sandbox_file_t context on added file
   shell: "stat /var/{{ vfp_dirname | quote }}/{{ vfp_filename | quote}} | grep svirt_sandbox_file_t"
```

Comment:
You can use `stat -c '%C' /path/to/file` to avoid the grep, and that would make it easier to register it as a variable, then see whether or not it's in a set/list.